docs/docsite/rst/playbook_guide/playbooks_variables.rst

```diff
@@ -331,14 +331,15 @@ When passing variables with ``--extra-vars``, you must escape quotes and other s
     ansible-playbook arcade.yml --extra-vars '{"name":"Conan O'\\\''Brien"}'
     ansible-playbook script.yml --extra-vars "{\"dialog\":\"He said \\\"I just can\'t get enough of those single and double-quotes"\!"\\\"\"}"
 
-If you have a lot of special characters, use a JSON or YAML file containing the variable definitions.
+If you have a lot of special characters, use a JSON or YAML file containing the variable definitions. Prepend with `@`.
```

Comment:
Can you move this line to the subheading below where it talks about using a json or yaml file and has the example with the @?